pkgs/top-level/all-packages.nix

```diff
@@ -803,6 +805,10 @@ with pkgs;
 
   sea-orm-cli = callPackage ../development/tools/sea-orm-cli { };
 
+  vcpkg-tool = callPackage ../development/tools/vcpkg-tool { fmt = fmt_10; };
+
+  vcpkg = callPackage ../development/tools/vcpkg {  };
```

Comment:
Since we are now in the by-name era, can you move this file `development/tools/vcpkg/default.nix` to `by-name/vc/vcpkg/package.nix`?

```suggestion
```